cli/cmd/vuln_host_show_assessment.go

```diff
@@ -121,6 +137,13 @@ Grab a CVE id and feed it to the command:
 					Value:      evalGUID,
 				})
 
+				// filter by collector_type
+				filter.Filters = append(filter.Filters, api.Filter{
+					Expression: "eq",
+					Field:      "evalCtx.collector_type",
+					Value:      vulCmdState.CollectorType,
+				})
+
 				cli.StartProgress(
 					fmt.Sprintf("Fetching vulnerabilities from host evaluation '%s'...", evalGUID),
```

Comment:
At this point we know what the collector type is, can we add it in the spinner like
```suggestion
					fmt.Sprintf("Fetching vulnerabilities from host evaluation '%s' (collector_type: %s) ...", evalGUID),
```

Reply:
Also, there is another line above with an info message:
```go
				cli.Log.Infow("latest assessment found", "eval_guid", evalGUID)
```

could we change it to
```go
				cli.Log.Infow("latest assessment found", "eval_guid", evalGUID, "collector_type", vulCmdState.CollectorType)
```